_includes/init-note.html

```diff
@@ -0,0 +1,6 @@
+<p class="site-note">
+  <strong>Note:</strong> We recommend loading <code>uswds-init.js</code> when using banner, header, or modal components to assist in preventing flashes of unstyled content (FOUC) as well as cumulative layout shift (CLS). This small JavaScript file (less than 1 KB minified) helps the browser know if the USWDS JavaScript library is loading properly.
+  <br>
+  <br>
+  To add <code>uswds-init</code> to your site, simply copy <code>uswds-init.min.js</code> from <code>dist/js</code> into your site assets and reference the file in the <code>&lt;head&gt;</code> of your HTML files. Alternatively, you can inline its contents directly into a <code>&lt;script&gt;</code> tag in your HTML files.
```

Comment:
Read through it again and I am wondering if users might get confused about where to find the `dist/js` directory. I wrote up a suggestion that might make it a little clearer and give another option, depending on how users have installed USWDS. What do you think?
```suggestion
  To add <code>uswds-init</code> to your site, simply copy <code>uswds-init.js</code> into your site assets from either the <a href="https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/js"><code>uswds-core/src/js</code></a> package or the <code>dist/js</code> directory in the <a href="{{  site.baseurl }}/download/">downloadable package</a>. Then, reference the file in the <code>&lt;head&gt;</code> of your HTML files. Alternatively, you can inline its contents directly into a <code>&lt;script&gt;</code> tag in your HTML files.
```